pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -93,9 +93,8 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 
 		// A weight of zero is not permitted, plugins can be disabled explicitly
 		// when configured.
-		f.pluginNameToWeightMap[name] = int(pg[name].Weight)
-		if f.pluginNameToWeightMap[name] == 0 {
-			f.pluginNameToWeightMap[name] = 1
+		if pg[name].Weight != 0 {
```

Comment:
We want the weight to be explicitly initialized to 1 when not set.

Reply:
Get it now. I thought we need users to set it explicitly. If so, we do not need to check the score plugins' weight, because it must have a value.

Reply:
Addressed. PTAL